tools/deploy/validator_test.go

```diff
@@ -193,12 +193,6 @@ func TestValidatorForLeaderFollowerSuccess(t *testing.T) {
 	require.NoError(t, validator())
 }
 
-type capturingValidator struct {
```

Comment:
is this change related?

Reply:
Not directly related, one of the deployment planner tests was flaky so this was just to fix that flaky test.